easybuild/easyconfigs/i/IMPUTE2/IMPUTE2-2.3.0-x86_64_dynamic.eb

```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = "Tarball"
+
+name = 'IMPUTE2'
+version = '2.3.0'
+versionsuffix = '_x86_64_dynamic'
+
+homepage = 'http://mathgen.stats.ox.ac.uk/impute/impute_v2.html'
+description = """  IMPUTE version 2 (also known as IMPUTE2) is a genotype imputation 
+ and haplotype phasing program based on ideas from Howie et al. 2009 """
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['https://mathgen.stats.ox.ac.uk/impute/']
+sources = ['%s_v%s%s.tgz' % (name.lower().rstrip('2'), version, versionsuffix)]
+
+sanity_check_paths = {
+    'files': ["impute2"],
+    'dirs': ["Example"]
+}
+
+# add the top dir to PATH
+modextrapaths = {
+    'PATH': ''
```

Comment:
OK, if `eb` notices this isn't a list, it'll make it a list and then continue, see https://github.com/hpcugent/easybuild-framework/blob/master/easybuild/framework/easyblock.py#L881

So, I guess this is fine.